frontend/package.json

```diff
@@ -31,7 +32,9 @@
     "enzyme-to-json": "^3.3.5",
     "eslint-plugin-import": "^2.18.2",
     "license-checker": "^25.0.1",
+    "prettier": "^1.19.1",
```

Comment:
You should pin prettier to exact version as KFP, because there can be formatting change between minor versions too.

Reply:
Done!